lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -84,11 +84,11 @@
 
   // TYPE: confirm
   .msg-error {
-    border-top: 2px solid $error-color;
-    background: tint($error-color,95%);
+    border-top: 2px solid $red1;
```

Comment:
@error-color is used in many places throughout the product, so I didn't feel comfortable changing it. There is a "newer" block of colors that had $red1, so I switched to that. I was also able to make the background slightly darker (90% tint) without running afoul of contrast violations-- to me, that looks better.

Note though the issue I already mentioned-- $warning-color is being used for "warning" messages, but it's a dark red (magenta) color, which is odd.